packages/analytics/analytics-chart/src/composables/useExploreResultToDatasets.spec.ts

```diff
@@ -26,6 +26,46 @@ describe('useVitalsExploreDatasets', () => {
     expect(result.value).toEqual({ datasets: [], labels: [] })
   })
 
+  it('can handle null metric values', () => {
+    const exploreResult: ComputedRef<AnalyticsExploreResult> = computed(() => ({
+      records: [{
+        version: 'v1',
+        timestamp: '2022-01-01T01:01:02Z',
+        event: {
+          dimension: null,
+        },
+      }],
+      meta: {
+        start: 1640998862,
+        end: 1640998870,
+        granularity: 5000,
+        dimensions: { dimension: ['dimension'] },
```

Comment:
The array maps to the actual values of that particular dimension that appears in the records so this should be 
`dimensions: { dimension: [null] },`

If there was another record with event 
```
       event: {
          dimension: 'service1',
        }
``` 
For example, then the meta would be 
`dimensions: { dimension: [null, 'service1'] },`